deliver/lib/deliver/commands_generator.rb

```diff
@@ -106,7 +120,7 @@ def run
         c.description = "Downloads existing metadata and stores it locally. This overwrites the local files."
 
         c.action do |args, options|
-          options = FastlaneCore::Configuration.create(Deliver::Options.available_options, options.__hash__)
+          options = FastlaneCore::Configuration.create(deliverfile_options(true), options.__hash__)
```

Comment:
Let's use named parameters to make it easier to understand when reading the code